app/dashboard/helpers.py

```diff
@@ -505,9 +528,13 @@ def process_bounty_changes(old_bounty, new_bounty):
     else:
         event_name = 'unknown_event'
         logging.error(f'got an unknown event from bounty {old_bounty.pk} => {new_bounty.pk}: {json_diff}')
-    
+
     print(f"- {event_name} event; diff => {json_diff}")
 
+    # record a useraction for this
+    record_user_action(event_name, old_bounty, new_bounty)
+
+
```

Comment:
Can we drop the second new line here?

Reply:
Can you rebase with master? I pushed a change to `sync_profile` last night that uses `update_or_create` and returns the profile.

Reply:
9284b2fb024e9b312447929ef7c08a170c85ba7a